### PR TITLE
Fix bugzilla41209

### PR DESCRIPTION
**Pull request by rmarinho**
### Description of Change ###

This issue seems to be a bug on android, for some reason the SeekBar gets the pressed state if we have a GestureDetector above on the hierarchy and the background ins transparent, i have checked and even with the correct Clickable = false; or DuplicateParentStateEnabled set to false this behaviour still occurs. 

My change is to override when android is trying to set the SeekBar as pressed state and check if we are or not touching the Slider itself before setting the pressed state.

The testing has to be with visual inspection . 

### Bugs Fixed ###

https://bugzilla.xamarin.com/show_bug.cgi?id=41029

### API Changes ###

None

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [x] Consolidate commits as makes sense

### CODE REVIEW

File: Xamarin.Forms.Platform.Android/Renderers/FormsSeekBar.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+			get
+			{
+				return base.Pressed;
+			}
+			set
+			{
+				if (isTouching)
+				{
+					base.Pressed = value;
+					isTouching &= value;
```
Comment (jassmith) [inline Xamarin.Forms.Platform.Android/Renderers/FormsSeekBar.cs]: The & seems to do nothing here.

**Comment by samhouts** [comment]
@rmarinho Please review Jason's comment above & rebase. :)

File: Xamarin.Forms.Platform.Android/Renderers/FormsSeekBar.cs (hunk 1 above)
Comment (rmarinho) [inline Xamarin.Forms.Platform.Android/Renderers/FormsSeekBar.cs]: true!  i think was XS suggestion

**Comment by rmarinho** [comment]
@samhouts done!

**Pull request merged**